cleanlab/multiannotator.py

```diff
@@ -318,6 +302,7 @@ def get_label_quality_multiannotator(
 def get_majority_vote_label(
     labels_multiannotator: Union[pd.DataFrame, np.ndarray],
     pred_probs: np.ndarray = None,
+    raise_dataset_errors: bool = True,
```

Comment:
Potentially change to `verbose`. Seems strange to have such a specific boolean for this.